### PR TITLE
(maint) Unit test fixes

### PR DESCRIPTION
**Pull request by willmeek**
Fixes for the unit tests. This also contains rubocop tidyup.

### CODE REVIEW

**Comment by DavidS** [comment]
The last rubocop complaint can be disabled, as `get_` has a specific meaning not connected to the rubocop rule

File: .travis.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ matrix:
       rvm: 2.4.4
     -
       env: PUPPET_GEM_VERSION="~> 4.0" CHECK=parallel_spec
-      rvm: 2.1.9
+      rvm: 2.3.1
```
Comment (DavidS) [inline .travis.yml]: puppet4 is shipping with ruby 2.1.9, so we need to test against that ruby version

**Review by DavidS — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (willmeek) [reply .travis.yml]: reverted

**Review by DavidS — APPROVED** [review]

**Comment by DavidS** [comment]
That looks like a common error on puppet4 - is this module up-to-date with the latest pdk templates?

**Comment by willmeek** [comment]
> That looks like a common error on puppet4 - is this module up-to-date with the latest pdk templates?

I don't recall PDKing it, I'll run that and get a seperate commit up

**Review by DavidS — APPROVED** [review]

**Pull request merged**